portfolio/src/main/webapp/script.js

```diff
@@ -115,28 +116,90 @@ function createCommentElement(comment) {
 }
 
 /** Tells the server to delete the task. */
-function deleteTask(comment) {
+function deleteTask(commentId) {
   const params = new URLSearchParams();
-  params.append('id', comment.id);
+  params.append('id', commentId);
   fetch('/delete-comment', {method: 'POST', body: params});
+  window.location.replace("index.html");
 }
 
-/* When the user clicks on the button, 
-toggle between hiding and showing the dropdown content */
+/**Onclicking the commentSettigns button show the dropdown-content */
 function commentSettings() {
-  document.getElementById("myDropdown").classList.toggle("show");
+  document.getElementById("dropdown-content").style.visibility = "visible"; 
+  document.getElementById("dropdown-content").style.display = "inline-block"; 
+}  
+
+/**Reload along with posting the maxComments to url, when user selects the no. of comments he wants to see. */
+function sendForm() {
+  window.location.replace("index.html?maxComments="+ document.getElementById('quantity').value );
+}
+
+/**Set the default value to maxComments when user doesn't give any input or when the page first loads. */
+function getMaxComments() {
+    const defaultValue = 15;
+    const queryString = window.location.search;
+    const urlParams = new URLSearchParams(queryString);
```

Comment:
Nit: no need to use 2 lines here:
` const urlParams = new URLSearchParams(window.location.search);`

Reply:
Done